### PR TITLE
update known issue status for 1.20 changelog

### PR DESCRIPTION
**Pull request by pacoxu**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:
The known issue is fixed now for #96873. it make sense for readers. 
**Which issue(s) this PR fixes**:

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.20.md [hunk 1]
```diff
@@ -2388,4 +2388,4 @@ filename | sha512 hash
 - github.com/godbus/dbus: [ade71ed](https://github.com/godbus/dbus/tree/ade71ed)
 - github.com/xlab/handysort: [fb3537e](https://github.com/xlab/handysort/tree/fb3537e)
 - sigs.k8s.io/structured-merge-diff/v3: v3.0.0
-- vbom.ml/util: db5cfe1
\ No newline at end of file
+- vbom.ml/util: db5cfe1
```
Comment (hasheddan) [inline CHANGELOG/CHANGELOG-1.20.md:2391]: Is this diff here intentional?

Comment (pacoxu) [reply CHANGELOG/CHANGELOG-1.20.md:2391]: sorry for that. As I edit the file on phone.

Comment (pacoxu) [reply CHANGELOG/CHANGELOG-1.20.md]: @hasheddan fixed now.

**Review by cpanato — APPROVED** [review]
/lgtm

**Review by hasheddan — APPROVED** [review]
/lgtm

**Pull request merged**